plugins/destination/plugin_testing.go

```diff
@@ -17,9 +17,26 @@ type PluginTestSuite struct {
 	tests PluginTestSuiteTests
 }
 
+// type MigrateResult int
+
+// const (
+// 	MigrateResultSuccess MigrateResult = iota
+// 	MigrateResultDropTable
+// )
+
+// MigrateStrategy defines which tests we should include
+// true means test should succeed for either automigrate
+type MigrateStrategy struct {
+	AddColumn           specs.MigrateMode
+	AddColumnNotNull    specs.MigrateMode
+	RemoveColumn        specs.MigrateMode
+	RemoveColumnNotNull specs.MigrateMode
+	ChangeColumn        specs.MigrateMode
+}
```

Comment:
It might be just semantics, but I'm a bit confused by the options here 🤔 

The table in the PR description has two options for each entry: `Auto Migrate / Drop Table`. I think it would be clearer if we used that terminology here, rather than `MigrateMode = forced` or `MigrateMode = safe`. `forced/safe` doesn't convey the same meaning to me `drop/auto`, but I think that is the intention of this configuration, right?

Reply:
Yeah I had the same dilemma if to introduce another type, but then it would be quite annoying as we will have `OverwriteForceStrategy` `OverwriteAutoMigrateStrategy` and the same for `append` so I preferred for now just to do one for each mode and then we can understand if it is being dropped or not.

But I think that yes, we potentially will need to change it, I just didn't want to make it even more complex right now as I think we will need to learn a bit more about this API and usage and then do another iteration on the testing API at least.

Reply:
Maybe a comment would help for now. Something like `forced == drop table`, `safe == do not drop table`